dev-requirements.txt

```diff
@@ -61,9 +61,11 @@ docutils>=0.13.1,<0.17; python_version == '2.7'
 docutils>=0.13.1; python_version >= '3.5' and python_version <= '3.9'
 docutils>=0.14; python_version >= '3.10'
 sphinx-git>=10.1.1
-# GitPython 3.0.0 removed support for Python 2.7
-GitPython>=2.1.1,<3.0.0; python_version == '2.7'
-GitPython>=2.1.1; python_version >= '3.5'
+# GitPython version 3.0.0 and newer does not support python 2.7
+# GitPython version 3.1.24 requires python_version GE '3.7'' 
+# GitPython version 3.1.27 fixes safety issue #52518 
+GitPython>=2.1.1<3.0.0; python_version <= '3.5'
+GitPython>=3.1.27; python_version >= '3.6'
```

Comment:
If GitPython version 3.1.24 requires python_version>='3.7' as the comment says, why does version 3.1.27 work on Python 3.6?

Reply:
DONE in my commit on this PR.

Reply:
OK with me